notesapi/v1/tests/test_views.py

```diff
@@ -430,10 +430,16 @@ def test_search(self):
         self.assertEqual(last_note, note)
         self.assertEqual(results['total'], 3)
 
-        results = self._get_search_results(text="Second")
-        self.assertEqual(results['total'], 1)
-        self.assertEqual(len(results['rows']), 1)
-        self.assertEqual(results['rows'][0]['text'], 'Second note')
+        def test_search(searchText, expectedText, expectedTags):
```

Comment:
I think this should have a different name from the encapsulating method to avoid confusion.